### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @apirakas

### DISCUSSION THREAD

**Discussion by apirakas** [heig-vd-dai-course/heig-vd-dai-course#35] [Show and tell]
[My first java project !](https://github.com/apirakas/java-intellij-idea-and-maven)

**Comment by ludelafo** [comment]
Thank you!

I would use code blocks (with ```` ```sh ```` for Shell scripts) to improve the reading of the commands.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
